src/CFITSIO.jl

```diff
@@ -2324,37 +2324,35 @@ for (a, b) in ((:fits_create_binary_tbl, 2), (:fits_create_ascii_tbl, 1))
 
             # get length and convert coldefs to three arrays of Ptr{Uint8}
             ntype = length(coldefs)
-            GC.@preserve coldefs begin
-                ttype = [pointer(x[1]) for x in coldefs]
-                tform = [pointer(x[2]) for x in coldefs]
-                tunit = [pointer(x[3]) for x in coldefs]
-                status = Ref{Cint}(0)
+            ttype = [x[1] for x in coldefs]
+            tform = [x[2] for x in coldefs]
+            tunit = [x[3] for x in coldefs]
```

Comment:
For what is worth, these could also be `getindex.(coldefs, 1)` etc., instead of using comprehension.